packages/button/Button/styles.scss

```diff
@@ -674,3 +687,188 @@ $caButton-verticalPaddingForm: calc(
     background: $color-red-100 !important; /* stylelint-disable-line declaration-no-important */
   }
 }
+
+%caButtonLabel {
```

Comment:
Seems that there's a few things here that have been duplicated in this file before the stuff that you added.

Reply:
Updated